### ISSUE TITLE
When using mouse scroll in inventory, precise crouch also happens, but also kinda slow, also GUI is huge there!

### ISSUE THREAD

**Issue by GreenboxGames** [naelstrof/KoboldKare#119]
Probably need a method to turn off the precise crouch in all areas that use the valuable mouse wheel control now and in the future. The wheel probably could use to scroll more but maybe just make the inventory and text like 6 times shorter? All of the GUI seems way too huge in the game ATM.

**Comment by naelstrof** [comment]
Yeah agreed, the UI in general is really really rough. It's about to get rougher with modding support.
